api/ccapi/include/layer.h

```diff
@@ -311,6 +321,14 @@ BackboneTFLite(const std::vector<std::string> &properties = {}) {
   return createLayer(LayerType::LAYER_BACKBONE_TFLITE, properties);
 }
 
+/**
+ * @brief Helper function to create Embedding layer
+ */
+inline std::unique_ptr<Layer>
+Embedding(const std::vector<std::string> &properties = {}) {
```

Comment:
Because this is a default layer, how about registering explicitly from `app_context.cpp@add_default_object`.

Reply:
done.